shopfloor/migrations/13.0.1.1.0/pre-migration.py

```diff
@@ -0,0 +1,77 @@
+from psycopg2 import sql
+
+from odoo.tools import column_exists
+
+
+def migrate(cr, version):
```

Comment:
FTR probably you could use openupgradelib instead of writing queries yourself

Reply:
The option to change the type of the field with `USING` to change the type and the value as I did doesn't exist.
Anyway, this is disposable code, doesn't really matter if there is repetition.

Reply:
Good point about `USING`, renaming field is there tho. Anyway my comment is not about repetition, is just about the possibility to save time/code if possible. :wink: 
Thanks for the feedback!